cluster/README.md

```diff
@@ -61,3 +61,49 @@ To reset your environment variables:
 ```shell
 source unset_common.sh
 ```
+
+### Building the images, an overview
+
+The docker images are built from the Makefiles inside the subdirectories here.
+The images have various entrypoints into the varioius minigo binaries depending
+on their purpose.  Some of the images only differ in the way the minigo cc/main
+binary is called, e.g. enabling TPU or not, etc.
+
+The images used are:
+
+  - `cc-base`: A base image with tensorflow precompiled with CUDA support.
+  - `minigo-cc-player`: A selfplay image using the C++ engine, built on the base
+    image, using GPUs by default.
+  - `minigo-tpu-player`: A selfplay image using the C++ engine, built to use
+    TPUs.
+  - `minigo-cc-evaluator`: An image built to run a game between two models,
+    saving the SGF to GCS, using the C++ engine (GPUs supported)
+  - `minigo-gpu-evaluator`: Same, but using the Python engine.
+  - `minigo-player`: Python selfplay (with compiled tensorflow wheel in python,
+    optimized for a target architecture, *not* using GPUs)
+
+The engines using the C++ engine (`cc-player`, `cc-evaluator`, `tpu-player`) are
+all built off of the cc-base image.
+
+In order to build an image, `cd` to the appropriate directory, read the Makefile
+to determine the appropriate targets, and note which environment variables are
+set.  In particular, `make` will automatically tag the built images so they can
+be uploaded to the Google Container Registry, so the environment variables
+identifying the project (`$PROJECT`) and the tag (`$VERSION_TAG`) need to be
```

Comment:
What should `VERSION_TAG` be set to? Is it just an arbitrary version number?

Reply:
yep.  they're like mpm labels -- totally optional and you can use them in whatever way keeps things organized.  I'll clarify the phrasing.